src/cockpit/channels/__init__.py

```diff
@@ -35,6 +36,7 @@
     InternalMetricsChannel,
     NullChannel,
     PackagesChannel,
+    PcpMetricsChannel,
```

Comment:
So merely importing this is enough to magically make the bridge use this, and ignore the old C bridge "pcp" manifest? I would feel better about it if this PR would already drop pkg/pcp/manifest.json. Dropping the rest of the cockpit-pcp code is fine in a follow-up, of course.

Reply:
Please drop the complete manifest, not just the `bridges` part. There's no need for having just that stub file.

Reply:
But the manifest has a requires version for cockpit-bridge in there? Should we just drop that?

Reply:
What's that good for? We are not expecting to ever actually call `cockpit-pcp` any more, or am I missing something? (If we do, then we still need the `bridges` bit)

Reply:
I thought the pages are under pcp but they are under metrics, nevermind. Dropped locally, will be removed on next push.